app/javascript/widget/components/ArticleCardSkeletonLoader.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="py-4 space-y-4 bg-white">
```

Comment:
@nithindavid The skeleton loader background color is white in dark mode

<img width="447" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/fe43dcf9-2784-484f-bc05-85eccf43d154">

```suggestion
  <div class="py-4 space-y-4 bg-white dark:bg-slate-700">
```